pkg/process/metadata/workloadmeta/grpc.go

```diff
@@ -0,0 +1,26 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package workloadmeta
+
+type mockableGrpcListener interface {
+	writeEvents(procsToDelete, procsToAdd []*ProcessEntity)
+}
+
+var _ mockableGrpcListener = (*grpcListener)(nil)
```

Comment:
If you need to stub out the listener then it would be better to just create a No-Op as a stub. The mockable is more for testing which this is not.